### PR TITLE
sig-storage: add COSI repos

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/org/issues/1942

/assign @saad-ali @xing-yang @jsafrane @msau42 
for lgtm

/hold
for lgtm from sig storage leads

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -2027,6 +2027,10 @@ sigs:
     - name: sig-storage-test-failures
       description: Test Failures and Triage
   subprojects:
+  - name: cosi
```
Comment (xing-yang) [inline sigs.yaml:2030]: @saad-ali Do we want this subproject to be called kubernetes-cosi or just cosi?
We have subproject named kubernetes-csi on line 2046.  Should this one be consistent and be called kubernetes-cosi?

Comment (wlan0) [reply sigs.yaml:2030]: Yeah, I would prefer kubernetes-cosi as well

Comment (nikhita) [reply sigs.yaml]: Updated to `kubernetes-cosi`

**Comment by xing-yang** [comment]
LGTM

**Comment by xing-yang** [comment]
We have this line  "this is a new subproject for sig-storage called kubernetes-cosi" written in the issue kubernetes/org#1942, so this PR should be fine.

**Pull request merged**